x-pack/plugins/enterprise_search/public/applications/workplace_search/index.tsx

```diff
@@ -17,13 +20,13 @@ import { Overview } from './components/overview';
 export const WorkplaceSearch: React.FC = () => {
   const { enterpriseSearchUrl } = useContext(KibanaContext) as IKibanaContext;
   return (
-    <>
+    <Provider store={store}>
```

Comment:
is there some way we can pass store the `http` lib in here on init so that we don't need to pass it to our actions? 🤔

Reply:
That would be the best case scenario but I am not sure how

Reply:
I can take a look but not totally sure when since I'm out tomorrow, probably sometime closer to Friday? FWIW, not a blocker for merge IMO

Reply:
That would be great. I'm going to write some tests anyway and get it into a mergeable state so I'll ping you when you get back and its ready.